cms/src/main/java/uk/nhs/digital/externalstorage/resource/ImageDisplayPlugin.java

```diff
@@ -0,0 +1,104 @@
+package uk.nhs.digital.externalstorage.resource;
+
+import org.apache.wicket.behavior.AttributeAppender;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.panel.Fragment;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.hippoecm.frontend.editor.compare.StreamComparer;
+import org.hippoecm.frontend.model.IModelReference;
+import org.hippoecm.frontend.plugin.IPluginContext;
+import org.hippoecm.frontend.plugin.config.IPluginConfig;
+import org.hippoecm.frontend.plugins.standards.util.ByteSizeFormatter;
+import org.hippoecm.frontend.service.IEditor;
+import org.hippoecm.frontend.service.render.RenderPlugin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+
+public class ImageDisplayPlugin extends RenderPlugin<Node> {
+
+    private static final long serialVersionUID = 1L;
+
+    private static final Logger log = LoggerFactory.getLogger(ImageDisplayPlugin.class);
+
+    public static final String MIME_TYPE_HIPPO_BLANK = "application/vnd.hippo.blank";
+
+    ByteSizeFormatter formatter = new ByteSizeFormatter();
```

Comment:
Can be private too. Following the doubt voiced before, I've checked that making this private doesn't fail compilation.